src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/ApiShould.cs

```diff
@@ -1787,6 +1790,302 @@ public void TestSqlCommandSequentialAccessCodePaths(string connection, CommandBe
             }
 
         }
+        [ConditionalTheory(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringSetupForAE))]
+        [ClassData(typeof(AEConnectionStringProvider))]
+        public void TestExecuteXmlReader(string connection)
+        {
+            IList<object> values = GetValues(dataHint: 60);
```

Comment:
Please add `CleanUpTable(connection, tableName);` before running the test case because we will use the same table again when targeting AAS-VBS. HGS-VBS and AAS-VBS are now sharing one sql server.